department-of-veterans-affairs/caseflow
# JHoang/APPEALS-22481

Resolves [24481](https://jira.devops.va.gov/browse/APPEALS-24481)

# Description
Fixed incomplete wording from banner when editing a single issue with MST and PACT to no special issue designation.

## Acceptance Criteria

## Testing Plan
1. Login as Intake Admin
2. Start intake process
3. Add a single issue with MST and PACT designated to the issue
4. Establish appeal
5. Go to the case details page for that appeal
6. Click on "Correct Issues" on the Issues section
7. Edit the single issue and remove both MST and PACT from the issue.
8. Press Save
9. Observe banner shows correct message on updates to MST and PACT designations

# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---
<img width="581" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/111081469/274bcedb-5d9a-4688-81da-e431e5dfbdb0">|<img width="569" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/111081469/60c7cc7f-7d49-4005-bb9a-b5d158f78612">

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
